app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -45,7 +45,7 @@ def pending_eps?
   end
 
   def rrd_pdf_created?
-    form.dig('rrd_metadata', 'pdf_created')
+    form.dig('rrd_metadata', 'pdf_created') || false
```

Comment:
Return `false` when `nil` would normally be returned so that calling this method shows "false" in the notification instead of empty string.